backend/src/data/migrations/20220808115524_add_rest_permissions.ts

```diff
@@ -0,0 +1,24 @@
+import { Knex } from 'knex';
+
+const TABLE_NAME = 'permissions';
+
+const permissions = [
+  { name: 'Manage Interviews', key: 'manage_interviews' },
+  { name: 'Manage Categories', key: 'manage_categories' },
+  { name: 'Manage Mentoring', key: 'manage_mentoring' },
```

Comment:
```suggestion
  { name: 'Manage Interviews', key: 'manage_interviews' },
    { name: 'Manage Interviews', key: 'manage_interview' },
  { name: 'Manage Categories', key: 'manage_categories' },
  { name: 'Manage Mentoring', key: 'manage_mentoring' },
```
you need to add one more